bundles/org.openhab.binding.weatherunderground/src/main/java/org/openhab/binding/weatherunderground/internal/handler/WeatherUndergroundHandler.java

```diff
@@ -553,7 +551,9 @@ private boolean updateWeatherData(Set<String> features) {
             }
 
             urlStr = urlStr.replace("%QUERY%", StringUtils.trimToEmpty(config.location));
-            logger.debug("URL = {}", urlStr);
+            logger.debug("URL = {}", urlStr.replace("%APIKEY%", "***"));
+
+            urlStr = urlStr.replace("%APIKEY%", StringUtils.trimToEmpty(bridgeHandler.getApikey()));
```

Comment:
We are currently in the process of removing use of apache commons.lang from addons, so please implement `trimToEmpty` by hand here.

Reply:
I will do it in a separate PR.
As you can see in my current PR, this line is not new but just moved.
Just changing this line would have no sense as there are several similar lines to be changed.

And I consider this PR as of a higher priority (security).